src/main/java/seedu/address/model/Model.java

```diff
@@ -6,13 +6,14 @@
 import javafx.collections.ObservableList;
 import seedu.address.commons.core.GuiSettings;
 import seedu.address.model.person.Person;
+import seedu.address.model.session.Session;
 
 /**
  * The API of the Model component.
  */
 public interface Model {
     /** {@code Predicate} that always evaluate to true */
-    Predicate<Person> PREDICATE_SHOW_ALL_PERSONS = unused -> true;
+    Predicate<Person> PREDICATE_SHOW_ALL_SESSION = unused -> true;
```

Comment:
`Model` seems to be used by the student list as well, should `person` be overridden to `session` or should a new one be added for `session`?

Reply:
The new changes keep both now!